pom.xml

```diff
@@ -44,8 +44,7 @@
         <axonserver.api.version>2023.0.1</axonserver.api.version>
 
         <!-- For compatibility between grpc and tcnative, check https://github.com/grpc/grpc-java/blob/master/SECURITY.md#netty -->
```

Comment:
I think this means we can get rid of this comment too, right?

Reply:
Yes, indeed.